bumble/smp.py

```diff
@@ -715,6 +718,21 @@ def check_expected_value(self, expected, received, error):
             return False
         return True
 
+    def prompt_user_for_confirm(self, next_steps):
```

Comment:
should probably be called `prompt_user_for_confirmation`.

Reply:
Done :+1: